internal/rukpak/convert/registryv1_test.go

```diff
@@ -438,50 +445,68 @@ func TestRegistryV1SuiteGenerateErrorAllNamespaceDisabled(t *testing.T) {
 	t.Log("By creating a registry v1 bundle")
 	watchNamespaces := []string{""}
 	unstructuredSvc := convertToUnstructured(t, svc)
-	registryv1Bundle := RegistryV1{
+	registryv1Bundle := convert.RegistryV1{
 		PackageName: "testPkg",
 		CSV:         *csv,
 		Others:      []unstructured.Unstructured{unstructuredSvc},
 	}
 
 	t.Log("By converting to plain")
-	plainBundle, err := Convert(registryv1Bundle, installNamespace, watchNamespaces)
+	plainBundle, err := convert.Convert(registryv1Bundle, installNamespace, watchNamespaces)
 	require.Error(t, err)
 	require.Nil(t, plainBundle)
 }
 
-func TestRegistryV1SuiteGeneratePropagateCsvAnnotations(t *testing.T) {
```

Comment:
Removed `TestRegistryV1SuiteGeneratePropagateCsvAnnotations` since it was testing an unexported function and the behavior is also tested on line 489